source/style-guide/markup/headings.txt

```diff
@@ -73,19 +73,19 @@ To avoid errors in your header markup syntax, stick to the following rules:
             ----------------
 
      * -
-         This example assumes the source constant ``{+Db+}`` resolves to 
+         This example assumes the source constant ``{{+db-example+}}`` resolves to 
```

Comment:
Cool, didn't you could format source constants like this!

Reply:
Strange that this page doesn't seem to exist in the left nav. I guess we should add ensuring the pages are all surfaced in the navigation to our list of tasks.